src/main/java/edu/kis/powp/jobs2d/drivers/adapter/MyAdapter.java

```diff
@@ -5,33 +5,46 @@
 import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.jobs2d.Job2dDriver;
 
+import edu.kis.powp.jobs2d.features.DrawerFeature; // added
+
 /**
  * driver adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
-	private int startX = 0, startY = 0;
+public class MyAdapter extends DrawPanelController implements Job2dDriver
```

Comment:
Class name not fixed!
Inheritance is not used (no super drawLine calls). Should be removed!